Tribler/Main/vwxGUI/CreditMiningPanel.py

```diff
@@ -0,0 +1,533 @@
+"""
+This module contains credit mining panel and list in wx
+
+Written by Ardhi Putra Pratama H
+"""
+
+import logging
+import wx
+from binascii import hexlify
```

Comment:
my IDE already gives me this import order. Tried some manual ordering but not successful yet.